<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Projectdlg_General</class>
 <widget class="QWidget" name="Projectdlg_General">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Project name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Project directory</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLineEdit" name="lePath"/>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="btnChange">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Architecture</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Family</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QComboBox" name="cmbArch">
     <item>
      <property name="text">
       <string>Picoblaze</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="cmbFamily">
     <item>
      <property name="text">
       <string>kcpsm1cpld</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>kcpsm1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>kcpsm2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>kcpsm3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>kcpsm6</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QTextEdit" name="teNotes">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QLineEdit" name="leName"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
